### ISSUE TITLE
Site level information is missing

### ISSUE THREAD

**Issue by Robinlovelace** [cyipt/actdev-ui#64]
In previous versions of the tool clicking on a site polygon showed a wealth (or rather excess) of info. Now the problem is the opposite: clicking on a site shows no additional data fields, as far as I can tell.

**Comment by mvl22** [comment]
No, that was in the study area region, which you wanted removed.

**Comment by mvl22** [comment]
No, that was in the study area region layer, which was removed.

**Comment by Robinlovelace** [comment]
Can that info be added to the site polygon popup?

**Comment by mvl22** [comment]
This has been added in dfbba472095ef0340f66903f68b07b5e7d4443fe.

However, the dictionary is proving very difficult to get working, so this may have to do for now. The site polygon derives from the region-switcher, and so is not a real layer in the same way that the others are. This means the standard mechanics for pumping in the dictionary aren't there, and I'm having trouble emulating this.

![Screenshot 2021-03-11 at 19 31 07](https://user-images.githubusercontent.com/361423/110843582-5e18e480-82a0-11eb-8d59-84fc404c2e36.png)

**Comment by Robinlovelace** [comment]
Looking good, certainly good enough for the demo test, great to have all that info and not a problem that it looks a bit rough around the edges. Great work!
